manifests/server/instance/service.pp

```diff
@@ -42,6 +42,10 @@
       status    => $service_status,
     }
 
+    Anchor["postgresql::server::service::begin::${name}"]
```

Comment:
I may deeply regret asking when I do not have a full background on this but my understanding was Anchor was a redundant pattern as the contain function came in?

Reply:
Yes anchors are from the past.  Not sure if attempting to remove them in this PR will help however (and that would be a backwards incompatible change).  But in the long term, having them gone is probably a good idea.